prow/config.yaml

```diff
@@ -19926,13 +19923,11 @@ periodics:
       - name: ssh
         mountPath: /etc/ssh-key-secret
         readOnly: true
-      - name: docker-graph
-        mountPath: /docker-graph
       # TODO(bentheelder): sort out container port
       ports:
       - containerPort: 9999
         hostPort: 9999
```

Comment:
I probably don't need the port anymore?

Reply:
yeah, as long as you aren't using the cache. can't see that on github though :man_shrugging: